### PR TITLE
add fitbit display data to physiome

### PR DESCRIPTION
**Pull request by shad0wfax**

### CODE REVIEW

File: src/controllers/physiomeController.js [hunk 1]
```diff
@@ -6,11 +6,25 @@ import { render } from '@doc.ai/neuron-app';
 import ensureArray from '../helpers/ensureArray';
 
 export const extractPhysiomeData = userData =>
```
Comment (even-steven) [inline src/controllers/physiomeController.js]: This isn't needed anymore, right? I guess it doesn't hurt to leave it in.

File: src/views/physiome.ejs [hunk 2]
```diff
@@ -1,42 +1,94 @@
...
+          </Row>
+        </Table>
+      </Card>
+    <% } -%>
+  <% } -%>
 
-  <% if (heartRate) { -%>
-    <Card title="Most recent Heart Rate reading" color={['#f5515f', '#9f041b']}>
+  <% if (fitbit) { -%>
+    <Card title="My Activity (Fitbit)" color={['#f5515f', '#9f041b']}>
```
Comment (even-steven) [inline src/views/physiome.ejs]: Not sure if you wanted to include a date range for the step summary (we do this in healthkit). ```<Text textAlign="center" paddingBottom="25"><%= healthkit.stepsDateRange %></Text>```

File: src/controllers/physiomeController.js (hunk 1 above)
Comment (shad0wfax) [reply src/controllers/physiomeController.js]: This is still required - the omicsController calls this method. It tries to figure out if we have at least one "physiome" data. This is determined by selecting if there is either of healthkit and/or fitbit with data.

Comment (even-steven) [reply src/controllers/physiomeController.js]: Ah, gotcha.

**Pull request merged**